src/content/structured/components/text-input/guidance.mdx

```diff
@@ -74,6 +74,8 @@ Use text fields as part of a form or other data collection page for capturing te
 
 Use clear labels to indicate what information should be entered into the field.
 
+When using min or max props, ensure to add visual indication of these values in the label or helper text.
```

Comment:
"When setting minimum and maximum values using the `min` and `max` props, make sure to add an indication of these formatting rules in the helper text."

Reply:
This is actually better palced in the next section for 'helper text'.